src/core/renderer.js

```diff
@@ -716,7 +716,7 @@ Renderer.prototype.refresh = function (timestamp) {
     this.computePool = [];
 
     tiles.forEach(t => {
-        if (t.style.getColor().isAnimated() || t.style.getWidth().isAnimated()) {
+        if (t.style.isAnimated()) {
```

Comment:
this loop is legacy code and should probably be removed since the requestAnimationFrame logic is not going to be on the core renderer

Reply:
:eyes: